templates/security/esm.html

```diff
@@ -8,318 +8,555 @@
 
 {% block content %}
 
-<section class="p-strip--suru-topped is-bordered">
-  <div class="row u-equal-height">
-    <div class="col-8">
-      <span class="u-sv3"><h1>Extended Security Maintenance</h1></span>
-      <span class="u-sv3"><p class="p-heading--4">Security updates for Ubuntu LTS for additional 5 years</p></span>
-      <p>Continue to receive security updates for the Ubuntu base OS, critical software packages and infrastructure components with Extended Security Maintenance (ESM). ESM provides five additional years of security maintenance, enabling an organization's continuous vulnerability management.</p>
-      <p>ESM is available through an Ubuntu Advantage for Infrastructure subscription for physical servers, virtual machines, containers and desktops, and is free for personal use. ESM is also included in the Ubuntu Pro premium images that are optimised for <a href="/cloud/public-cloud">the public cloud</a>, and these also provide security maintenance for high and critical CVEs for the entire collection of software packages shipped with Ubuntu.</p>
-      <p>
-        <a href="/pro" class="p-button--positive">Get started with Ubuntu Advantage</a>
-        <a href="/security/esm#get-in-touch" class="p-button">Contact us</a  >
-        </p>
+<section class="p-strip--image is-deep" style="background-image:url('https://assets.ubuntu.com/v1/18717572-05_suru2_light_glow_8K_edited.png'); background-position: 70% bottom;">
```

Comment:
We should make this into a standalone class, ex. p-strip--suru-light-glow to avoid using inline styles and keep future uses of it consistent.